exercises/practice/majority-element/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Find the most repeated word in the encyclopedia",
```

Comment:
```suggestion
  "blurb": "Find the most repeated color of candy in a bag",
```

Reply:
Good catch! This was from an earlier version. I'll update this locally as it should say "majority element" instead of "most repeated."